conf.d/project/etc/test-config.json

```diff
@@ -15,8 +15,8 @@
         "info": "Launch all the tests",
         "action": "lua://AFT#_launch_test",
         "args": {
-            "trace": "hello",
-            "files": ["helloworld.lua"]
+            "trace": "AFT",
```

Comment:
Why change ? If you don't trace "hello" api you will not be able to see its events coming.